File: common/src/main/java/dev/number6/comprehend/results/PresentableEntityResults.java
package dev.number6.comprehend.results;

import java.time.LocalDate;
import java.util.Map;

public class PresentableEntityResults extends ComprehensionResults<Map<String, Map<String, Long>>> {

    public PresentableEntityResults(LocalDate comprehensionDate, Map<String, Map<String, Long>> entitiesResults, String channelName) {
        super(comprehensionDate, entitiesResults, channelName);
    }

    @Override
    public String toString() {
        return "EntityResults{" +
                "comprehensionDate=" + getComprehensionDate() +
                ", entitiesResults=" + getResults() +
                ", channelName='" + getChannelName() + '\'' +
                '}';
    }
}

File: common/src/main/java/dev/number6/comprehend/results/PresentableEntityResults.kt
package dev.number6.comprehend.results

import java.time.LocalDate

class PresentableEntityResults(comprehensionDate: LocalDate,
                               entitiesResults: Map<String, Map<String, Long>>,
                               channelName: String) :
        ComprehensionResults<Map<String, Map<String, Long>>>(comprehensionDate, entitiesResults, channelName) {
    override fun toString(): String {
        return "EntityResults{" +
                "comprehensionDate=" + comprehensionDate +
                ", entitiesResults=" + results +
                ", channelName='" + channelName + '\'' +
                '}'
    }
}